src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -710,10 +710,14 @@ ucs_status_t uct_dc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
         goto out;
     }
 
-    if ((flags & UCT_FLUSH_FLAG_REMOTE) &&
-        ucs_test_all_flags(ep->flags, UCT_DC_MLX5_EP_FLAG_FLUSH_RKEY |
-                                      UCT_DC_MLX5_EP_FLAG_FLUSH_REMOTE)) {
-        return uct_dc_mlx5_ep_flush_remote(ep, comp);
+    if (flags & UCT_FLUSH_FLAG_REMOTE) {
+        UCT_CHECK_PARAM(uct_rc_iface_is_flush_remote(&iface->super.super),
+                        "iface %p is not configured to process flush(remote) "
```

Comment:
`"dc_iface %p on %s: flush(remote) is not supported", iface, dev_name)`

Reply:
> dc_iface %p on %s: flush(remote) is not supported

ok, reworded